packages/db/tests/test-connection.js

```diff
@@ -135,14 +135,8 @@ test('getKnexClient logs retry errors and throws expected knexTimeoutError', asy
     knex(t.context.tableName).where({}),
     { instanceOf: KnexTimeoutError }
   );
-  const actual = [loggerWarnStub.args[0][0], loggerWarnStub.args[0][1].message];
-  t.deepEqual(
-    actual,
-    [
-      'knex failed on attempted connection',
-      'connect ECONNREFUSED 127.0.0.1:5400',
-    ]
-  );
+  t.deepEqual(loggerWarnStub.args[0][0], 'knex failed on attempted connection');
+  t.true(loggerWarnStub.args[0][1].errors.map((e) => e.message).includes('connect ECONNREFUSED 127.0.0.1:5400'));
```

Comment:
This was changed as the underlying modules changed behaviors in the update and this can result in an aggregate error with more values than the original test.